frontend-react/package.json

```diff
@@ -150,7 +152,7 @@
     "camelcase": "^6.2.1",
     "case-sensitive-paths-webpack-plugin": "^2.4.0",
     "cross-env": "^7.0.3",
-    "css-loader": "^6.7.3",
+    "css-loader": "5.2.6",
```

Comment:
Hmmmm I don't think we should downgrade major versions specifically for @storybook/preset-scss -- these are loaders that we're using to build production code.

Reply:
agreed, I can revert this and see if it messes with anything.